src/ontology/uberon-edit.obo

```diff
@@ -222488,6 +222488,14 @@ relationship: dc-contributor http://orcid.org/0000-0003-2699-3066
 relationship: part_of UBERON:7500094 ! tubercle of calcaneus
 property_value: dcterms-date 2022-05-19T21:16:42Z xsd:dateTime
 
+[Term]
+id: UBERON:7500109
+name: basion
+def: "An anatomical point on the ventral border of foramen magnum in the median plane." [ISBN:9780873659505]
```

Comment:
I wasn't able to find an accessible version of the dbxref to cross check, but this definition seems like it would encompass two points on the ventral surface of the foramen magnum- both anterior and posterior, since the median plane would cross through both. Based on some quick searches, I think the basion refers only to the point on the anterior margin. https://radiopaedia.org/articles/basion

Reply:
that's correct, I'll update it to be more specific